api/leaderworkerset/v1/leaderworkerset_types.go

```diff
@@ -58,6 +58,18 @@ const (
 	// This will be applied to all API objects including:
 	// leaderStatefulset, leaderPods, workerStatefulsets, workerPods.
 	TemplateRevisionHashKey string = "leaderworkerset.sigs.k8s.io/template-revision-hash"
+
+	// Environment variable added to all containers in the LeaderWorkerSet to
+	// address the leader via the headless service.
+	LwsLeaderAddress string = "LWS_LEADER_ADDRESS"
+
+	// Containers requesting TPU resources will have the following environment
+	// variables added.
+	TpuWorkerHostNames string = "TPU_WORKER_HOSTNAMES"
```

Comment:
Sorry, but please don't this is vendor related, let's keep them under the accelerators where different accelerators hold their logics.